addons/binding/org.openhab.binding.eightdevices/ESH-INF/i18n/eightdevices_xx_XX.properties

```diff
@@ -0,0 +1,13 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
Šitą failą reikia sutvarkyti pagal reikalavimus: xx_XX pakeisti į en_US ir visus aprašymus sudėti pagal tai kokie yra daiktai ir kanalai